mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/domain/Costs.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2023 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.mantisrx.server.master.domain;
+
+import io.mantisrx.shaded.com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Value;
+
+@Value
+public class Costs {
```

Comment:
Add more comments to this.

Reply:
Is this class actually necessary? why do we need our own wrapper around a double?

Reply:
Today, this covers the total cost. Imagine we need to break this down further into cpuCosts, memCosts, diskCosts, etc... Having this value class will help wrap all that without adding more fields to the JobMetadata.

Reply:
OK, that's a fair point.